README.md

```diff
@@ -1,4 +1,69 @@
-### d8 build scripts
+## d8 build scripts
 
 The repository is part of the [Compiler Explorer](https://godbolt.org/) project. It builds
 the docker images used to build the d8 runtime/compiler/tooling used on the site.
+
+### Using the Dockerfile to make a d8 build locally
+
+You need to have docker installed on your machine. If you do not have docker installed you can
+get it from [here](https://docs.docker.com/engine/install/)
+
+After doing this, run the following command in the root directory of this repository:
+
+```
+docker build -t d8:trunk .
+```
+This will create a container in your machine named `d8:trunk`
+
+You can check if the container has been built by running the following command:
+```
+docker images
+```
+You should see the image listed as follows:
+```
+REPOSITORY   TAG        IMAGE ID       CREATED        SIZE
+d8           trunk      fcd7f69f0e42   3 weeks ago    8.31GB
+```
+
+Once you have this, you can run a container with this image using the following command:
+```
+docker run -d -t d8:trunk
+```
+
+You can see the running container by running the following command:
+```
+docker ps
+```
+
+You will see something like this:
+```
+CONTAINER ID   IMAGE      COMMAND   CREATED              STATUS              PORTS     NAMES
+cfdab51b3818   d8:trunk   "fish"    About a minute ago   Up About a minute             compassionate_lamport
+```
+
+You can then open a shell inside this container using:
+```
+docker exec -it cfdab51b3818 fish
```

Comment:
There is no `fish` inside this container. (It's my shell of choice too, but I don't put it into general purpose things; it's enough having to expalin to folks how all this stuff works without giving them a new shell to contend with too!)

Reply:
Yes that makes sense. I see what happened here. I used my own image instead of the this container. I'll change this to sh or bash.